deploy/helm/kubecf/values.yaml

```diff
@@ -310,3 +301,31 @@ k8s-host-url: ""
 k8s-service-token: ""
 k8s-service-username: ""
 k8s-node-ca: ""
+
+dependencies:
+  - name: percona-xtradb-cluster
+    enabled: true
+    repository: file://percona-xtradb-cluster
+
+# Configuration for the embedded database solution using the PXC sub-chart.
+percona-xtradb-cluster:
+  replicas: 1
+
+  image:
+    repository: "splatform/percona-xtradb-cluster"
+    tag: 0.0.2
+
+  ssl:
+    enabled: true
+    secret: kubecf.var-pxc-certs
```

Comment:
This is incorrect. The secret name uses a dynamic value. It's coincidental that it works if the deployer specifies `kubecf` as the installation name using Helm. Is it possible to automatically set this value?

Reply:
This is fixed by https://github.com/SUSE/kubecf/pull/393/commits/4328e181004a5668963f03d3b7c5fdc8742c4441.